docs/readme_link.md

```diff
@@ -0,0 +1,2 @@
+```{include} ../README.md
```

Comment:
what if we called this URL something like `/getting-started`?

Reply:
I added a separate Getting Started section.